packages/oauth-client/src/EarthdataLoginClient.ts

```diff
@@ -53,7 +53,7 @@ export class EarthdataLoginClient extends OAuthClient {
       default:
         return new EarthdataLoginError(
           'UnexpectedResponse',
-          `Unexpected response: ${httpError.response.body}`
+          `Unexpected response: ${JSON.stringify(httpError.response.body)}`
```

Comment:
should we just use httpError.response.body.error?

Reply:
This error is not specific, so we want detailed error message.
I got "[Object, Object]" in one of the integration-test run before the fix, and still don't the detailed error is.